### ISSUE TITLE
diabetes_prediction_KNN

### ISSUE THREAD

**Issue by MaryamSabourii** [MaryamSabourii/diabetes_prediction#1]
import pandas as pd
import numpy as np
from sklearn.model_selection import train_test_split
from sklearn.neighbors import KNeighborsClassifier
from sklearn.metrics import accuracy_score
from sklearn.preprocessing import StandardScaler

#download the diabetes dataset CSV file from Kaggle
diabetes_dataset = pd.read_csv('diabetes.csv') 

x = diabetes_dataset.drop(columns='Outcome' , axis=1)
y = diabetes_dataset['Outcome']

scaler = StandardScaler()
scaler.fit(x)
x_scaled = scaler.transform(x)
x = x_scaled

x_train, x_test, y_train, y_test = train_test_split(x_scaled, y, test_size = 0.2)

# Create a KNN classifier object
model = KNeighborsClassifier(n_neighbors =60)

model.fit(x_train, y_train)
#model.score(x_test, y_test)

input_data = (2,197,70,45,543,30.5,0.158,53)
input_data_as_numpy_array = np.asarray(input_data)

# reshape the array as we are predicting for one instance
input_data_reshape = input_data_as_numpy_array.reshape(1, -1)

# standardize the input data
std_data = scaler.transform(input_data_reshape)

ypred = model.predict(std_data)

print(ypred)

if ypred[0]==0:
    print('The person is not diabetic')
elif ypred[0]==1:
    print('The person is diabetic')
